webpack/scenes/FlatpakRemotes/FlatpakRemotesPage.js

```diff
@@ -0,0 +1,148 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import { Link } from 'react-router-dom';
+import { Table, Thead, Th, Tbody, Tr, Td } from '@patternfly/react-table';
+import TableIndexPage from 'foremanReact/components/PF4/TableIndexPage/TableIndexPage';
+import {
+  useSetParamsAndApiAndSearch,
+  useTableIndexAPIResponse,
+} from 'foremanReact/components/PF4/TableIndexPage/Table/TableIndexHooks';
+import { useTableSort } from 'foremanReact/components/PF4/Helpers/useTableSort';
+import EmptyPage from 'foremanReact/routes/common/EmptyPage';
+import Pagination from 'foremanReact/components/Pagination';
+import { translate as __ } from 'foremanReact/common/I18n';
+import { STATUS } from 'foremanReact/constants';
+import { selectFlatpakRemotes, selectFlatpakRemotesError, selectFlatpakRemotesStatus } from './FlatpakRemotesSelectors';
+import { urlBuilder } from '../../__mocks__/foremanReact/common/urlHelpers';
+import { truncate } from '../../utils/helpers';
+
+const FlatpakRemotesPage = () => {
+  const response = useSelector(selectFlatpakRemotes);
+  const error = useSelector(selectFlatpakRemotesError);
+  const status = useSelector(selectFlatpakRemotesStatus);
+
+  const {
+    results = [], subtotal, page, per_page: perPage,
+  } = response || {};
+
+  const columnHeaders = [__('Name'), __('URL'), __('Last Scan')];
+  const COLUMNS_TO_SORT_PARAMS = {
+    [columnHeaders[0]]: 'name',
+    [columnHeaders[1]]: 'url',
+    [columnHeaders[2]]: 'lastScan',
+  };
+
+  const apiOptions = { key: 'FLATPAK_REMOTES' };
+  const defaultParams = { page, perPage };
+
+  const apiResponse = useTableIndexAPIResponse({
+    apiUrl: '/katello/api/v2/flatpak_remotes',
+    apiOptions,
+    defaultParams,
+  });
+
+  const { pfSortParams } = useTableSort({
+    allColumns: columnHeaders,
+    columnsToSortParams: COLUMNS_TO_SORT_PARAMS,
+    initialSortColumnName: 'Name',
+  });
+
+  const actionsWithPermissions = () => [
+    { title: __('Scan'), isDisabled: true },
+    { title: __('Edit'), isDisabled: true },
+    { title: __('Delete'), isDisabled: true },
+  ];
+
+  const {
+    setParamsAndAPI,
+    params,
+  } = useSetParamsAndApiAndSearch({
+    defaultParams,
+    apiOptions,
+    setAPIOptions: apiResponse.setAPIOptions,
+  });
+
+  const onPaginationChange = (newPagination) => {
+    setParamsAndAPI({
+      ...params,
+      ...newPagination,
+    });
+  };
+
+  return (
+    <TableIndexPage
+      apiUrl="/katello/api/v2/flatpak_remotes"
+      apiOptions={{ key: 'FLATPAK_REMOTES' }}
+      header={__('Flatpak Remotes')}
+      createable={false}
```

Comment:
```suggestion
      creatable={false}
```
This is misspelled in Foreman, so needs to be this way here. (Or feel free to improve Foreman..)

Reply:
I have implemented the suggested change